spec/models/item_spec.rb

```diff
@@ -5,4 +5,14 @@
     expect(create(:item)).to be_valid
   end
 
+  it 'has a status of OOS if condition is OOS' do
+    item = build :item, condition: 'Out of Service', status: 'Unassigned'
+    expect(item.status).to eq('Out of Service')
```

Comment:
this just seems flat-out wrong. You told it to build you an item with a status of "Unassigned" but then you get a status of "Out of Service" if this passes something is _seriously_ wacky with how this works, and would be very confusing to any developers who aren't you.